.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: 'We really appreciate your effort to provide feedback. Before opening a new
+  issue, please make sure that this case is not already reported in GitHub as an
+  issue or in StackOverflow as a question.'
+
+---
+
+**Environment**
```

Comment:
Can we add `nativescript-angular` and `angular/*` versions here?

Reply:
Totally, check if it's good now.

Reply:
:+1: